arrows/mvg/camera_options.h

```diff
@@ -0,0 +1,138 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief Header for MVG camera options.
+ */
+
+#ifndef KWIVER_ARROWS_MVG_CAMERA_OPTIONS_H_
+#define KWIVER_ARROWS_MVG_CAMERA_OPTIONS_H_
+
+#include <arrows/ceres/lens_distortion.h>
+#include <arrows/mvg/kwiver_algo_mvg_export.h>
+
+#include <vital/config/config_block.h>
+#include <vital/types/camera_map.h>
+#include <vital/types/camera_perspective.h>
+#include <vital/vital_types.h>
+
+#include <unordered_map>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace mvg {
+
+/// various models for lens distortion supported in the config
+enum LensDistortionType
+{
+  NO_DISTORTION,
+  POLYNOMIAL_RADIAL_DISTORTION,
+  POLYNOMIAL_RADIAL_TANGENTIAL_DISTORTION,
+  RATIONAL_RADIAL_TANGENTIAL_DISTORTION
+};
+
+/// Provide a string representation for a LensDisortionType value.
+KWIVER_ALGO_MVG_EXPORT
+const char*
+LensDistortionTypeToString(LensDistortionType type);
+
+/// Parse a LensDistortionType value from a string or return false.
+KWIVER_ALGO_MVG_EXPORT
+bool
+StringToLensDistortionType(std::string value, LensDistortionType* type);
+
+/// Default implementation of string options for cam enums
+template < typename T >
+std::string
+mvg_options()
```

Comment:
I know this is just copied from ceres, but I can't remember why this template needs to be exposed in the header.  It seems like it is only used in the cxx.

Reply:
that's for the declarative part of MVG_ENUM_HELPERS